src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+    private Car car;
+    private static final int ATTEMPTS = 5;
+
+    @BeforeEach
+    public void setUp() {
+        car = new Car(ATTEMPTS);
+    }
+
+    @Test
+    @DisplayName("moves 배열 검증")
+    public void generateMoves() {
+        car.generateMoves();
+        assertThat(car.getMoves().length).isEqualTo(ATTEMPTS);
```

Comment:
```suggestion
        assertThat(car.getMoves()).hasSize(ATTEMPTS);
```
hasSIze 를 사용해 볼 수도 있을 거 같아요 😄